UI/window-basic-settings.hpp

```diff
@@ -247,9 +247,13 @@ class OBSBasicSettings : public QDialog {
 	void LoadServices(bool showAll);
 	void OnOAuthStreamKeyConnected();
 	void OnAuthConnected();
+	void FetchUserInfo();
+	void FetchUserInfoFinished();
 	QString lastService;
 	int prevLangIndex;
 	bool prevBrowserAccel;
+	QPointer<QThread> fetchInfoThread;
```

Comment:
Did you mean to use QScopedPointer here?

Reply:
Using a `QScopedPointer` here would lead to a crash/exception. And I must admit I'm not entirely sure why.